### PR TITLE
test: upgrade to jest 28

### PR DESCRIPTION
**Pull request by nicolas377**

### CODE REVIEW

**Comment by nicolas377** [comment]
I'm waiting on ts-jest to release v28 to latest.

**Comment by nicolas377** [comment]
This PR does need to be broken up into several different PRs. This PR will be closed, but a few more PRs will be opened.

File: package.json [hunk 1]
```diff
@@ -60,20 +63,22 @@
...
+    "jest-puppeteer": "~6.1.0",
     "postcss": "~8.4.13",
     "prettier": "~2.6.2",
+    "puppeteer": "~13.7.0",
     "react": "~18.1.0",
     "react-dom": "~18.1.0",
     "standard-version": "~9.3.2",
     "tailwindcss": "~3.0.24",
-    "ts-jest": "~27.1.4",
+    "ts-jest": "~28.0.0-next.3",
```
Comment (nicolas377) [inline package.json]: Forgot to upgrade to 28.0.0 on latest instead of next

**Review by Guy-Adler — APPROVED** [review]

**Comment by nicolas377** [comment]
Landed in 6edefe68978689141d2d0d17e668f606789568fc

**Pull request closed without merging**